python/pyspark/sql/tests/test_dataframe.py

```diff
@@ -144,6 +144,17 @@ def test_drop_duplicates(self):
             message_parameters={"arg_name": "subset", "arg_type": "str"},
         )
 
+    def test_drop_duplicates_with_multi_args(self):
```

Comment:
Nit: I feel that this test name is not clear to tell this is to test `AMBIGUOUS_REFERENCE`.

I guess either rename it to something like `test_drop_duplicates_with_ambiguous_reference` or add a comment like `# SPARK-XXXX: test drop API when there are ambiguous reference`?

Reply:
nice, updated